ToSidewalk/nodes.py

```diff
@@ -140,27 +122,14 @@ def get_shared_way_ids(self, other):
             return list(set(self.way_ids) & set(other.get_way_ids()))
 
     def get_sidewalk_nodes(self, wid):
-        """ Return sidewalk nodes
```

Comment:
Can you do the presence check like my version? (i.e., if wid in self.sidewalk_nodes. Also I want the docstring (comments) to be kept.